packages/client/src/router/loaders/redirectLoader.ts

```diff
@@ -0,0 +1,7 @@
+import { redirect } from 'react-router-dom'
+import { IUser } from '../../utils/scripts/api/types'
+
+const redirectLoader = (user: IUser | null | undefined) =>
+  user ? redirect('/') : null
```

Comment:
_Не критично к исправлению_

Тут немного нетипичная и сложно воспринимаемая функция. В случае наличия `user` ты вызываешь метод и "ничего" не возвращаешь (на самом деле undefined), а в случае отсутствия возвращаешь значение

Если тебе не критичен именно возврат null, то можно переписать на вариант, в котором не будет явного указания на `null` (возвращать функция будет `undefined` или `false`)

```suggestion
const redirectLoader = (user: IUser | null | undefined) => !!user && redirect('/');
```

или ещё более просто, но с чуть большим количеством кода через `if`

Reply:
В поиске по реквесту не нашел использования `redirectLoader`. Проверь дополнительно, актуальна ли эта функция